jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -177,6 +177,14 @@ public static void setServerInfo(String serverInfo)
         __serverInfo = serverInfo;
     }
 
+    public enum ContextStatus
```

Comment:
Not a big fan of the name `UNSET`.
An alternative would be to have
```java
    Boolean initialized; // null==not initialized; TRUE==initialized;  FALSE==destroyed 
```
Hmmm can't use a switch then... but maybe that's not necessary in most cases?

Perhaps just rename `UNSET` to `STOPPED` or `IDLE`  or  'NOT_INITIALIZED`

Reply:
But NOT_INITIALIZED is not the same as UNSET .... NOT_INITIALIZED could mean the same as DESTROYED, whereas UNSET is clearly not either of the other two.